src/hgraph/nodes/_compound_scalar_operators.py

```diff
@@ -33,7 +33,7 @@ def from_ts_node(**kwargs):
     return from_ts_node(**kwargs)
 
 
-@compute_node(resolvers={COMPOUND_SCALAR: lambda mapping, tsb: mapping[TS_SCHEMA].py_type.scalar_type()})
+@compute_node(all_valid=('tsb',), resolvers={COMPOUND_SCALAR: lambda mapping, tsb: mapping[TS_SCHEMA].py_type.scalar_type()})
```

Comment:
Is this really required? We only need all_valid if the nodes don't have defaults

Reply:
We're producing a scalar from a bundle - the standard data class init requires all non-defaulted fields to be provided otherwise it throws. An alternative would be to generate valid requirements for non-defaulted fields only but that's a bit trickier.